README.md

```diff
@@ -178,6 +179,24 @@ If you have direct access to the underlying Kubernetes Nodes in your cluster, CO
 
 **_TODO_** Port https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/754155622/CORTX+Kubernetes+N-Pod+Deployment+and+Upgrade+Document+using+Services+Framework#5.-Understanding-Management-and-S3-Endpoints-and-configuring-External-Load-balancer-service(Optional) here or into a linked `doc/readme` file.
 
+### Granular Deployment (Data Pod Only)
+
+Apart from the steps given in [Deploying CORTX on Kubernetes](#deploying-cortx-on-kubernetes) steps above, add following steps for granular data Pod only deployment.
+    
+1.  Update the Deployment Mode in solution configuration file like solution.yaml: 
+By default, Deployment Mode is set to standard 
+For data pod only deployment set the deployment_type to data-only
+    
+    ```bash
+    deployment_type: data-only
+    ```
+    
+   Currently framework support the following 2 modes:
+   - standard - Deploy all redefine pods
+   - data-only - Deploy only Data pod
+
+2.  Validate deployment, please note here you will have only Data Pod.
```

Comment:
Do we really need this line?  The main deploy documentation already describes how to validate the deployment.  Why add an additional step here?

Reply:
Removed 
I added this section to notify that in data only deployment we will see only data pod.
I think it is not required